docs/operations/recovery-from-permanent-quorum-loss-in-etcd-cluster.md

```diff
@@ -47,8 +48,23 @@ There are two [Etcd clusters](https://github.com/gardener/etcd-druid/tree/master
       Delete all PVCs that are attached to `etcd-main` cluster.
 
       `kubectl delete pvc -l instance=etcd-main`
-   5. Edit the `etcd-main` cluster's configmap (ex: `etcd-bootstrap-4785b0`) as follows:
-      
+
+   5. Check the etcd's member leases. There should be leases starting with `etcd-main` as many as `etcd-main` replicas.
+   One of those leases will have holder identity as `<etcd-member-id>:Leader` and rest of etcd member leases have holder identities as `<etcd-member-id>:Member`.
```

Comment:
Would it be better to re-write the sentence as:
```suggestion
   One of those leases will have holder identity as `<16 digit etcd member id>:Leader` and rest of etcd member leases have holder identities as `<16 digit etcd member id>:Member`.
```

I suggested this because at the first look I was trying figure out the meaning of `etcd`, `member`, `id`, instead of considering them as part of a single phrase.

Reply:
should we mention about `16 digit` 🤔 ?

Reply:
We can mention that if we are sure that all the member ids are 16 digits. Are  we limiting the holder identity to only contain 16 digits? If so, we can mention `16 digit` in the doc.

Reply:
but lease holder identity is already mentioned ? `16 digit` might cause an confusion and it is a miscellaneous information which need not to be present in doc

Reply:
Okay. I agree. We don't need to mention `16 digit` as it may add further confusion . I am resolving this conversation .